cli/src/ux/__tests__/ConnectionManager.test.ts

```
@@ -0,0 +1,300 @@
+    pid: 12345,
+    stdout: { pipe: jest.fn() },
+    stderr: { pipe: jest.fn() },
+    on: jest.fn((event: string, callback: (code?: number) => void) => {
+      listeners[event] = listeners[event] || [];
+      listeners[event].push(callback);
+      if (event === 'spawn') {
+        callback();
+      }
+    }),
+    kill: jest.fn(() => {
+      (listeners.exit || []).forEach((cb) => cb(0));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)